services/monitoring/grafana/scripts/import.py

```diff
@@ -139,14 +139,24 @@ def subsituteDatasources(
 
 def main(foldername: str = "", overwrite: bool = True):
     # Get mail adress for alerts:
-    grafanaAlertingMailTarget = env.str("GRAFANA_ALERTS_MAIL")
+    try:
+        grafanaAlertingMailTarget = env.str("GRAFANA_ALERTS_MAIL")
+    except EnvError:
+        grafanaAlertingMailTarget = None
```

Comment:
you could use a default here: see https://github.com/sloria/environs/blob/master/environs/__init__.py#L71

```suggestion
    grafanaAlertingMailTarget = env.str("GRAFANA_ALERTS_MAIL", default=None)
```

Reply:
will add thx